migrations/versions/rev-2020-06-30-20:07:55-f177c14710f1_.py

```diff
@@ -18,6 +18,9 @@
 
 def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
+    op.execute('DELETE FROM custom_forms WHERE deleted_at IS NOT NULL')
+    op.execute('DELETE FROM event_sub_topics WHERE deleted_at IS NOT NULL')
```

Comment:
Add equivalent statements in downgrade as well

Reply:
What will be the equivalent?